apps/database/src/models/GithubRepository.ts

```diff
@@ -62,7 +62,7 @@ export class GithubRepository extends Model {
           to: "github_installations.id",
         },
         modify(builder) {
-          return builder.findOne({ deleted: false });
+          builder.where("github_installations.deleted", false);
```

Comment:
It could return multiple activeInstallation. It not the expected result

Reply:
Nop, it is already a `hasOne` relation.

Reply:
I tested it.